openmicroscopy/openmicroscopy
# Remove old Importer scripts (See #9364)

Test case (Linux and Windows):
- Download the packaged OMERO.clients zip file,
- Verify that importer-gui.sh and importer-win.bat are **not** present
- Ultimately check that on every platform (Win, Linux, OS X) there is a way to start the new Importer
